pymc3/backends/ndarray.py

```diff
@@ -43,9 +43,9 @@ def setup(self, draws, chain):
             self.draws = old_draws + draws
             self.draws_idx = old_draws
             for varname, shape in self.var_shapes.items():
-                old_trace = self.samples[varname]
-                new_trace = np.zeros((draws, ) + shape, self.var_dtypes[varname])
-                self.samples[varname] = np.concatenate((old_trace, new_trace),
+                old_vtrace = self.samples[varname]
```

Comment:
yep, it is just the array of values for a single variable.  I couldn't think of a better name (maybe `vsamples` or `varsamps`?), but wanted to it clear that it wasn't a trace object.  Ideas?